examples/terraform/aws/main.tf

```diff
@@ -110,7 +110,7 @@ module "elb_k8s_elb" {
   source  = "terraform-aws-modules/elb/aws"
   version = "~> 4.0"
 
-  name    = "${var.cluster_name}-k8s-api"
+  name    = substr("${var.cluster_name}-k8s-api", 0, 32)
```

Comment:
How long are the cluster names?? :sweat_smile:

Reply:
`talos-e2e-6aeac191-aws-nonfree-nvidia-k8s-api` something like this from ci,